lib/cc/engine/analyzers/violation.rb

```diff
@@ -91,19 +112,15 @@ def fingerprint
           digest << "-"
           digest << current_sexp.mass.to_s
           digest << "-"
-          digest << occurrences.to_s
+          digest << occurrences.length.to_s
```

Comment:
Ok, thanks, @pbrisbin. That makes sense as a description of desired behavior. If that's the behavior we want, I think we have to rethink the fingerprinting here more -- right now fingerprints are different for each instance because the hash uses `current_sexp.file`.

Making it so that a new instance of the same duplication keeps the old fingerprint is an interesting wrinkle. Do you know how we did that in classic? I can't think of a solution that would be 100% reliable.